src/main/java/school/lemon/changerequest/java/FractionNumber.java

```diff
@@ -0,0 +1,109 @@
+package school.lemon.changerequest.java;
+
+/**
+ * Created by lera on 14.12.16.
+ */
+public class FractionNumber {
+    private int dividend;
+    private int divisor;
+
+    public FractionNumber(int dividend, int divisor) {
+        if (divisor == 0) throw new IllegalArgumentException("divisor must not be zero");
+        if (dividend == 0) {
+            this.dividend = dividend;
+            this.divisor = 1;
+        } else {
+            this.dividend = dividend;
+            this.divisor = divisor;
+            int nod = nod(this.dividend, this.divisor);
+            this.dividend = dividend / nod;
+            this.divisor = divisor / nod;
+        }
+
+    }
+
+    public int nod(int dividend, int divisor) {
```

Comment:
Should it be public ? do you have any requirements to create some public `nod` method ?